### ISSUE TITLE
changelog for releases

### ISSUE THREAD

**Issue by lentferj** [osxmidi/LinVst#58]
It would be very nice if you could add a "Changes" section in the release description to describe shortly what changes the latest release incorporate. The way you add commits to the git repository makes it quite hard to follow otherwise.

**Comment by osxmidi** [comment]
Yes, good idea.

**Comment by keybreak** [comment]
@osxmidi 
Please do this at least briefly with thesis, it's already necessity for some testing / professional production use cases.

I've already have big groups of plugins, which can be affected by some changes, example:
If i would knew that there are some changes in presets / parameters saving & restoring code, i would absolutely do test of 42 plugins that were previously known to be affected - to make sure that existing projects won't break on update.
